src/runtime/dom-particle.ts

```diff
@@ -148,7 +148,7 @@ export class DomParticle extends XenStateMixin(DomParticleBase) {
   }
 
   async onHandleUpdate({name}: Handle, {data, added, removed}): Promise<void> {
-    if (data) {
+    if (data !== undefined) {
```

Comment:
What condition is this catching?

Reply:
It literally says on the tin ... any value of data that is not strictly undefined is a legitimate update value.